msal/tests/Test.MSAL.NET.Integration/Test.MSAL.NET.Integration.csproj

```diff
@@ -4,6 +4,9 @@
     <TargetFramework>net4.5.2</TargetFramework>
 
     <IsPackable>false</IsPackable>
+
+    <SignAssembly>true</SignAssembly>
```

Comment:
Good idea, we should sign all our test assemblies instead of applying the sn hack

Reply:
can this be applied to all test and sample projects?

---
In reply to: [222932902](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1304#discussion_r222932902) [](ancestors = 222932902)